### ISSUE TITLE
API Backend Feedback form not submitting (after 1.3 upgrade)

### ISSUE THREAD

**Issue by brylie** [apinf/api-umbrella-dashboard#939]
While working on the Meteor 1.3.x upgrade, the API Backend Feedback form does not appear to be working.

Steps to reproduce:
1. Add an API backend
2. View the API backend page
3. Click the "Feedback" tab
4. Click "Add Feedback"
5. Fill in, and submit the form

# Related
Issue #935 - update Meteor

**Comment by bajiat** [comment]
@frenchbread Can you take a look?

**Comment by frenchbread** [comment]
@bajiat Self-assigned

**Comment by frenchbread** [comment]
Reopening this issue since it still exists:
```
Exception while invoking method 'sendEmail' RecipientError: Can't send mail - no recipients defined
```

**Comment by brylie** [comment]
@frenchbread were you submitting the 'Contact' form or 'API Backend -> Feedback' form? The above error seems more relevant to the 'Contact' form, which is on the front page near the bottom.

**Comment by frenchbread** [comment]
@brylie The bug was for the "Contact" form submit. Wrong issue..
Anyway the issue above it resolved with #970
